go.mod

```diff
@@ -163,3 +163,5 @@ require (
 replace github.com/docker/docker => github.com/docker/docker v20.10.3-0.20221013203545-33ab36d6b304+incompatible // 22.06 branch
 
 replace github.com/metal-stack/go-ipam => github.com/dave-tucker/go-ipam v0.0.0-20230220173413-c25c56b2408b
+
+replace github.com/redhat-et/go-oidc-agent => github.com/chirino/go-oidc-agent v0.0.0-20230308174912-b4dc11f2e6eb // addroutes branch
```

Comment:
Can you open a PR please in go-oidc-agent please? I'd rather that we didn't use `replace` unless we really have to.